python/common/python/math/math_gamma.java

```diff
@@ -0,0 +1,242 @@
+package python.gammafunction;
```

Comment:
The gamma and erf implementation are large in their own respect. I don't think its wise to include this in the main file. This package acts as helpers package.